src/components/pages/SystemPage.jsx

```diff
@@ -84,7 +84,7 @@ const SystemPage = (props) => {
         <Nav>
           {menu}
         </Nav>
-        <div className={`content ${isOverflow ? 'overflow-x-auto' : ''} pb-0`}>
+        <div className={`content min-h-0 ${isOverflow ? 'overflow-x-auto' : ''} pb-0`}>
```

Comment:
I think that you need to set the value to about 90vh so that the inscription below can be seen

Reply:
done